### DISCUSSION TITLE
Add ability to launch the dev environment with a normal domain, instead of localhost:8080

### DISCUSSION THREAD

**Discussion by Dalton123** [WeAreAthlon/frontend-webpack-boilerplate#193] [Ideas]
In the previous version of the boilerplate you could set it up like so
`external: {
    proxy: "http://example.test/",
  },`

This would change the URL whist developing. I can't figure out how to recreate this feature since updating to the latest code on the repo.

Does this feature still exist in webpack 5 and this boilerplate?

Thanks

**Comment by pnikolov** [comment]
@Dalton123 the server name and port can be configured in the `configuration/environment.js` file:

https://github.com/WeAreAthlon/frontend-webpack-boilerplate/blob/efb0a6c4085178e696c0bdb5ffe1e1a0526e18d6/configuration/environment.js#L11-L14
